<launch>
    <node
        name="detector_node"  pkg="picker" type="detector_node.py" output="screen" respawn="true">
    </node>
</launch>
